src/main/java/com/google/api/codegen/grpcmetadatagen/java/JavaGrpcPackageMetadataTransformer.java

```diff
@@ -22,7 +22,8 @@
   private final Map<String, String> snippetsOutput =
       ImmutableMap.of(
           "LICENSE.snip", "LICENSE",
-          "metadatagen/java/grpc/package.snip", "build.gradle");
+          "metadatagen/java/grpc/package.snip", "build.gradle",
```

Comment:
Can we rename the snip file to "build.gradle.snip"? It's a little annoying that the file name bears no resemblance to the output.